pkg/sql/logictest/REPOSITORIES.bzl

```diff
@@ -0,0 +1,46 @@
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# When updating `cockroach_releases.yaml`, update the _PREDECESSOR_VERSION
+# and the SHA256 sums in _CONFIGS.
```

Comment:
This requires an extra step in the release process or in https://github.com/cockroachdb/cockroach/blob/master/.github/workflows/update_releases.yaml. Should we create a template for this file or change the values in place? 🤔 
What happens if we forget updating these?

Reply:
The test will fail if `cockroach_releases.yaml` and this file are not in-sync.

Not sure what you mean by creating a template for this file. My thinking was the person doing the update would update these values in place.

Reply:
If you want I can extract this stuff (the SHA's and version number) into its own file without all the other stuff. Does that make things easier?

Reply:
> My thinking was the person doing the update would update these values in place.

I don't think people are manually updating `cockroach_releases.yaml`; the GitHub worklow mentioned by Rail is updating it automatically and opening PRs when there's a change.

Could we update [update_releases.go](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/release/update_releases.go) to automate this step as well?

Reply:
Done. I'll need someone to pay extra attention to `update_releases.go` to make sure I'm doing it right -- I'm not 100% sure I've implemented the logic correctly, though it seems to work for `master`.